packages/mongodb-chatbot-ui/src/App.tsx

```diff
@@ -38,6 +38,7 @@ function App() {
           name="MongoDB AI"
           shouldStream={shouldStream}
           darkMode={darkMode}
+          fetchOptions={{ credentials: "include" }}
```

Comment:
Should we add to DevCenterChatbot's wrapper below this too?

Reply:
yeah good catch